### PR TITLE
Scroll copied cards into view in card grid

### PR DESCRIPTION
**Pull request by habdelra**
This PR will scroll the copied card into view after it's been copied. 

scroll-copied-card

### CODE REVIEW

**Review by jurgenwerk — APPROVED** [review]
Looks good! Maybe some time later we can also make the newly created item pop out with a flashing border, like @FadhlanR did when jumping to the override card in schema editor: 

https://github.com/cardstack/boxel/assets/273660/dca96b1d-9369-4986-afdd-666397ba09a7

**Comment by habdelra** [comment]
> Looks good! Maybe some time later we can also make the newly created item pop out with a flashing border, like @FadhlanR did when jumping to the override card in schema editor:
> 
>  Screen.Recording.2023-10-19.at.09.05.20.mov

Yeah, I actually suggested that in the ticket. @lukemelia  said to just scroll for now.

File: packages/host/app/components/operator-mode/container.gts [hunk 1]
```diff
@@ -128,10 +131,7 @@ export default class OperatorModeContainer extends Component<Signature> {
     });
   }
 
-  get stacks() {
-    return this.operatorModeStateService.state?.stacks ?? [];
-  }
-
+  // public API
```
Comment (tintinthong) [inline packages/host/app/components/operator-mode/container.gts:134]: redundant comment?

**Comment by tintinthong** [comment]
From my testing, I noticed that if we had 2 of the same workspace side by side(both drafts). It wudn work. Is this something we want to support given users will likely use only their own workspace?

EDIT: It does seem real wrong but it seems (as a user) something I will actually do

File: packages/host/app/components/operator-mode/container.gts (hunk 1 above)
Comment (habdelra) [reply packages/host/app/components/operator-mode/container.gts:134]: I just wanted to make it clear that these are intentionally not private. I see that many times we have a habit of not being clear about what is private and what is not in our components, and that this is not that kind of situation

**Comment by habdelra** [comment]
> From my testing, I noticed that if we had 2 of the same workspace side by side(both drafts). It wudn work. Is this something we want to support given users will likely use only their own workspace?
> 
> EDIT: It does seem real wrong but it seems (as a user) something I will actually do

Good catch. I didn’t think about that situation. I can fix that in this pr

**Comment by tintinthong** [comment]
https://github.com/cardstack/boxel/assets/8165111/9a63dcaa-29a0-4ced-8e35-61a3e9ed938b

I also found an error state when opening "Add card" while in the process of copying but may not be directly related to this PR

**Comment by habdelra** [comment]
> Screen.Recording.2023-10-19.at.9.18.25.PM.mov 
> I also found an error state when opening "Add card" while in the process of copying but may not be directly related to this PR

i'll check it out

**Comment by habdelra** [comment]
I'm noticing a few other issues in this:
1. the velcro addon is not working after doing the copy (i'm guessing this was always the case). when you hover over a grid item nothing happens
2. the stack item header gets scrolled up (look closely at the screenshot in the description). I think this is a side effect of velcro using `visibility: hidden` instead of `display: hidden` to hide the items.

**Comment by habdelra** [comment]
> Screen.Recording.2023-10-19.at.9.18.25.PM.mov 
> I also found an error state when opening "Add card" while in the process of copying but may not be directly related to this PR

this is unrelated to copying. it seems to be a general problem with creating cards--you see this without the copy button too

create-bug

filed here: https://linear.app/cardstack/issue/CS-6163/bug-unable-to-create-new-cards

**Pull request merged**